src/seismometer/_api.py

```diff
@@ -130,7 +130,8 @@ def target_feature_summary(exclude_cols: list[str] = None, inline=False):
     """
     sg = Seismogram()
 
-    exclude_cols = exclude_cols or sg.entity_keys
+    exclude_cols = exclude_cols or [key for key in sg.entity_keys]
+    exclude_cols += sg.dataframe.select_dtypes(include=["datetime", "datetimetz", "datetime64[ns]"]).columns.tolist()
```

Comment:
I couldnt find a good changelog on ydataprofiling expectations, but regardless of specifics should this same thing be added to feature_summary ~50lines up for consistency?  

or looking at it more, maybe have a flag (default True) and do this as a straight `reduced_frame = df.select_dtypes(exclude=...)` in ComparisonReportWrapper and SingleReportWrapper?  
I'm not seeing anything requiring this be done first, and in fact, could possible craft a case where this removes a column and breaks the FilterRule

Reply:
moved into the report wrapper as a new function `filter_unsupported_columns`